src/createEslintConfig.ts

```diff
@@ -0,0 +1,74 @@
+import fs from 'fs';
+import path from 'path';
+import { CLIEngine } from 'eslint';
+
+interface CreateEslintConfigArgs {
+  rootDir: string;
+  writeFile: boolean;
+}
+export function createEslintConfig({
+  rootDir,
+  writeFile,
+}: CreateEslintConfigArgs): CLIEngine.Options['baseConfig'] {
+  const config = {
+    extends: [
+      'eslint:recommended',
+      'plugin:@typescript-eslint/recommended',
+      'prettier/@typescript-eslint',
+      'plugin:prettier/recommended',
+      'plugin:react/recommended',
```

Comment:
It'd be nice to have an override for `.tsx` files and extend the React specific plugins only for those files.  Would need https://github.com/eslint/eslint/pull/11554 to do that though.

Reply:
I think we just include the react config no matter what.